Source/WebCore/platform/graphics/gstreamer/GLVideoSinkGStreamer.cpp

```diff
@@ -79,6 +79,42 @@ static void initializeDMABufAvailability()
     });
 }
 
+#if !GST_CHECK_VERSION(1, 24, 0)
+static GstVideoFormat drmFourccToGstVideoFormat(uint32_t fourcc)
+{
+    switch (fourcc) {
+    case DRM_FORMAT_XRGB8888:
+        return GST_VIDEO_FORMAT_BGRx:
```

Comment:
```suggestion
        return GST_VIDEO_FORMAT_BGRx;
```
The same for every `return`.